src/component/common.js

```diff
@@ -22,7 +22,7 @@ import NotFoundPage from '../notFound'
 import { ComponentTabs } from './tabs'
 import { PersistentDrawerLeft } from '../layout'
 
-import GardenerLogo from '../resources/gardener-logo.svg'
+import ODG_Logo from '../resources/odg-logo.svg'
```

Comment:
nit: please use CamelCase

Reply:
sure, can do. my rationale behind using SK here was the fact that ODG is an acronym (hence requiring capital characters):
-> `ODGLogo` -> is slightly hard to read
-> `OdgLogo` -> is actually wrong, because ODG is an acronym
😞

Reply:
was thinking about the very same, but I think `ODGLogo` still fits better into codebase
also, I do **not** think readability really suffers here :-)